spyder_kernels/customize/tests/test_utils.py

```diff
@@ -16,7 +16,7 @@ def test_user_sitepackages_in_pathlist():
     """Test that we include user site-packages in pathlist."""
     if sys.platform.startswith('linux'):
         user_path = 'local'
-    elif sys.platform == 'darwin':
+    elif (sys.platform == 'darwin' or sys.platform.startswith('freebsd'):
```

Comment:
```suggestion
    elif (sys.platform == 'darwin' or sys.platform.startswith('freebsd')):
```

The initial parenthesis was not closed at the end of the statement.